### PR TITLE
chore: finish up add select testing

### PR DESCRIPTION
**Pull request by davidmenendez**
Contributes to #2124 

adds additional testing to finish up `AddSelect` coverage.

Screenshot 2022-07-22 at 16-11-01 Code coverage report for src_components_AddSelect

note- the coverage isn't 100% because an error caused by carbon is preventing me from creating passing tests

Screen Shot 2022-07-22 at 4 13 06 PM

### CODE REVIEW

File: packages/cloud-cognitive/src/components/AddSelect/AddSelectBody.test.js [hunk 1]
```diff
@@ -226,27 +268,45 @@ describe(componentName, () => {
...
       document.querySelectorAll('.bx--breadcrumb-item')[0].textContent
-    ).toBe('Categories');
-    fireEvent.click(childrenButton);
+    ).toBe('Business terms');
+    fireEvent.click(childrenBtn[0]);
     expect(document.querySelectorAll('.bx--breadcrumb-item').length).toEqual(2);
     expect(
       document.querySelectorAll('.bx--breadcrumb-item')[1].textContent
     ).toBe('California');
+    fireEvent.click(document.querySelectorAll('.bx--breadcrumb-item')[0]);
```
Comment (matthewgallo) [inline packages/cloud-cognitive/src/components/AddSelect/AddSelectBody.test.js:299]: We should probably be pulling in the Carbon prefix from:
```js
import { carbon } from '../../settings';

...
...
fireEvent.click(document.querySelectorAll(`.${carbon.prefix}--breadcrumb-item`)[0]);
...
...
```

Comment (matthewgallo) [reply packages/cloud-cognitive/src/components/AddSelect/AddSelectBody.test.js:299]: Same for the other instances of `.bx.....` within this file

Comment (davidmenendez) [reply packages/cloud-cognitive/src/components/AddSelect/AddSelectBody.test.js:299]: oh crap i forgot to replace the prefix's. thanks. will fix.

**Comment by davidmenendez** [comment]
@matthewgallo fixt

**Review by matthewgallo — APPROVED** [review]

**Pull request merged**